deploy/vagrant/conf/alluxio.yml

```diff
@@ -10,7 +10,7 @@ Github:
   Version: branch-1.6
 
 Release:
-  Version: 1.6.0 # name of release version
+  Version: 1.6.1 # name of release version
```

Comment:
@aaudiber is this intended?

Reply:
yes, 1.6.1 is the latest release version